app/models/note.rb

```diff
@@ -2,6 +2,8 @@ class Note < ActiveRecord::Base
   belongs_to :candidate
   belongs_to :user
   before_save :update_candidate
+  validates :comment, presence: true, length: { in: 10..2000 }
```

Comment:
My initial thought was to have this in a separate PR, but you're right...this shouldn't go out without the appropriate flash errors; failing silently is the worst.